[bug] Damon crashes when NOMAD_ADDR does not start with http://

Hello, thanks for making `damon`! It is a great TUI!

I came across this `nil pointer dereference` bug:

```bash
$ NOMAD_ADDR=192.168.1.13:4646 damon
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x55bf74812a97]

goroutine 1 [running]:
github.com/hcjulz/damon/nomad.(*Nomad).Namespaces(0xc00017fe30?, 0x55bf7495c1c0?)
        /home/h/.cache/paru/clone/damon-git/src/damon/nomad/namespaces.go:13 +0x17
main.initializeState(0x0)
        /home/h/.cache/paru/clone/damon-git/src/damon/cmd/damon/main.go:116 +0xeb
main.main()
        /home/h/.cache/paru/clone/damon-git/src/damon/cmd/damon/main.go:49 +0x13b
```

But if I use `NOMAD_ADDR=http://192.168.1.13:4646 damon` then it works as expected.

I'm new to Nomad and still learning about it, so I'm not sure what are all protocols supported to connect to a remote cluster. My suggestion is to issue a message/warning `missing protocol in nomad address. It should be one of http://, https://, etc://. More details in <link to docs>.` and gracefully terminate.

Also, when the port is missing we see no message about that, just that `damon` can't reach Nomad:

```bash
$ NOMAD_ADDR=http://192.168.1.13 damon
2023/10/18 12:54:22 cannot initialize view. Is Nomad running?
```

A suggestion to add the port in the address would be nice as well.

---

Some info on my system:

```bash
$ damon -v
Damon v0.1.0-dev (523ef4f)

$ nomad -v
Nomad v1.6.1

$ cat /etc/os-release
NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://bugs.archlinux.org/"
PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
LOGO=archlinux-logo
```

--- a/cmd/damon/main.go
+++ b/cmd/damon/main.go
@@ -44,7 +44,11 @@ func main() {
 		os.Exit(0)
 	}
 
-	nomadClient, _ := nomad.New(nomad.Default)
+	nomadClient, err := nomad.New(nomad.Default)
+	if err != nil {
+		fmt.Println("failed to generate Nomad client: ", err)
+		os.Exit(1)
+	}
 
 	state := initializeState(nomadClient)
 

